donkeycar/management/kivy_ui.py

```diff
@@ -853,14 +856,16 @@ def on_aug_list(self, obj, aug_list):
         if not self.config:
             return
         self.config.AUGMENTATIONS = self.aug_list
-        self.augmentation = ImageAugmentation(self.config, 'AUGMENTATIONS')
+        self.augmentation = ImageAugmentation(self.config, 'AUGMENTATIONS',
+                                              always_apply=True)
         self.on_current_record(None, self.current_record)
 
     def on_trans_list(self, obj, trans_list):
         if not self.config:
             return
         self.config.TRANSFORMATIONS = self.trans_list
-        self.transformation = ImageAugmentation(self.config, 'TRANSFORMATIONS')
+        self.transformation = ImageAugmentation(self.config, 'TRANSFORMATIONS',
```

Comment:
This doesn't look correct since transformations are no longer handled by ImageAugmention, but rather by ImageTransformations.  But I don't know what this method does; I don't see it called anywhere.

Reply:
Also there are now pre-augmentation and post-augmentation transformations.  So the complete list of transformations is given by 
```
transforms = getattr(cfg, 'TRANSFORMATIONS', []) + getattr(cfg, 'POST_TRANSFORMATIONS', [])
```

Reply:
`ImageAugmentation` is the class that performs the transformations. Did you rename it or create another one? This one works, I have tested it. But of course we don’t want to have two classes doing the same thing. Ill add the post transformations too.

Reply:
The point of my PR is to refactor the transformation pipeline so it can use standard parts and we can allow users to add a custom transform through configuration.  So to do this I removed transforms from ImageAugmentation and made a new class called ImageTransformations.  There are now two lists for transformations, one that runs prior to augmentation and one that runs post-augmentation.

Reply:
See https://github.com/DocGarbanzo/donkeycar/blob/1109-refactor-augmentations-dg/donkeycar/parts/image_transformations.py